style.css

```diff
@@ -0,0 +1,174 @@
+#container {
+  width: 375px;
+  height: 5410px;
+  background-color: #f7f7f9;
+}
+
+#template1 {
+  width: 375px;
+  height: 718px;
+}
+
+.toolbar {
+  width: 375px;
+  height: 92px;
+}
+
+.bar {
+  width: 375px;
+  height: 44px;
+  display: flex;
+}
+
+.bar > p {
+  width: 54px;
+  height: 16px;
+  margin-top: 16px;
+  margin-left: 21px;
+  font-family: 'poppins', sans-serif;
+  font-weight: 500;
+  text-align: center;
+  font-size: 14px;
+  line-height: 16.41px;
+  color: #000;
+  letter-spacing: -0.28px;
+}
+
+.mobile_network {
+  width: 17px;
+  height: 10.67px;
+  margin-left: 218.67px;
+  color: #000;
+  margin-top: 17.67px;
+}
+
+.mobile_wifi {
+  width: 15.27px;
+  height: 10.97px;
+  margin-top: 17.33px;
+  margin-left: 5.03px;
+  color: #000;
+}
+
+.mobile_battery {
+  width: 24.33px;
+  height: 11.33px;
+  margin-left: 5.03px;
+  margin-top: 17.33px;
+  border-radius: 1.33333px;
+}
+
+header {
+  width: 375px;
+  height: 44px;
+  display: flex;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  margin-top: 4px;
+  margin-left: 24px;
+  text-align: center;
+  font-family: 'poppins', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.Union {
+  width: 18px;
+  height: 12px;
+  margin-top: 6px;
+  margin-left: 223px;
+}
+
+.bg_image {
+  height: 10px;
+  width: 375px;
+  margin: auto;
+}
+
+.intro {
+  width: 327px;
+  height: 144px;
+  margin-top: 84px;
+  margin-left: 24px;
+  font-weight: 700;
+  font-size: 30px;
+  color: #172b4d;
+  line-height: 52px;
+  font-family: 'poppins', sans-serif;
+}
+
+.description {
+  width: 327px;
+  height: 146px;
+  margin-left: 24px;
+  margin-top: 24px;
+  font-family: 'poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  font-style: normal;
+  color: #344563;
```

Comment:
- [ ] Kindly make sure your page is not overflow on the right as you can see 
![Screenshot from 2022-09-22 20-39-42](https://user-images.githubusercontent.com/97100091/191826229-d437c8b5-232d-456c-8a38-40ff757b458c.png)
`hint: you can use the percentage (%) for your width instead of the fixed unit px`